www/package.json

```diff
@@ -4,6 +4,7 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+        "@auth0/auth0-react": "^1.5.0",
```

Comment:
Cool, is this like a special namespace for auth0?

Reply:
Specifically the `@auth0` part

Reply:
Looking at [auth0 - npm](https://www.npmjs.com/package/auth0) it might not be a "namespace" for their other libraries